js/renovation/ui/scroll_view.tsx

```diff
@@ -99,20 +122,21 @@ export default class ScrollView extends JSXComponent<ScrollViewPropsType>() {
   }
 
   @Method()
-  scrollBy(distance: number | Partial<Location>): void {
-    const { direction } = this.props;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Why do you use any but not ensureLocation parameter type.

Reply:
We have a restrictions in our generators - https://github.com/DevExpress/devextreme-renovation/issues/519